### PR TITLE
riemann: report flush errors with msg_error() instead of trace

### PR DESCRIPTION
**Pull request by bazsi**
Signed-off-by: Balazs Scheidler <bazsi77@gmail.com>

### CODE REVIEW

**Review by smortex — APPROVED** [review]
While the issue we try to diagnose does not reach this code, it still makes sense to log accordingly to the error status. LGTM!

File: modules/riemann/riemann-worker.c [hunk 1]
```diff
@@ -348,25 +348,41 @@ riemann_worker_flush(LogThreadedDestWorker *s, LogThreadedFlushMode mode)
                                                  MAX(1, owner->super.batch_lines));
   if (!r)
     {
+      msg_error("riemann: error calling riemann_communicate()",
+                evt_tag_str("server", owner->server),
+                evt_tag_int("port", owner->port),
+                evt_tag_int("batch_size", self->event.n),
+                evt_tag_int("ok", r->ok),
+                evt_tag_str("error", r->error),
```
Comment (smortex) [inline modules/riemann/riemann-worker.c:356]: In this codepath, `r` is `NULL`, so we can't access `ok` / `error` here.

Comment (bazsi) [reply modules/riemann/riemann-worker.c]: thanks for the note, I've fixed this.

**Comment by bazsi** [comment]
added news file, ready for review.

**Review by alltilla — APPROVED** [review]

**Pull request merged**